src/Containers/Nav/Nav.js

```diff
@@ -56,8 +58,26 @@ export default function ButtonAppBar(props) {
             </Button>
           ) : (
             <GoogleLogin
-              onSuccess={(credentialResponse) => {
-                localStorage.setItem(JWT_NAME, credentialResponse.credential);
+              onSuccess={async (credentialResponse) => {
+                const { credential } = credentialResponse;
+                const resp = await APIs.signAccount(credential);
+
+                if (resp.status !== 200) {
```

Comment:
not a blocker, but should we check for specific error status to give more meaningful message?

Reply:
If the `resp.response.data exists`, it will be shown on the snack bar. I think it's enough